src/AWS.Deploy.Common/ProjectDefinitionParser.cs

```diff
@@ -126,7 +126,9 @@ private async Task<bool> ValidateProjectInSolution(string projectPath, string so
 
             var lines = await _fileManager.ReadAllLinesAsync(solutionFile);
             var projectLines = lines.Where(x => x.StartsWith("Project"));
-            var projectPaths = projectLines.Select(x => x.Split(',')[1].Replace('\"', ' ').Trim()).ToList();
+            var projectPaths = projectLines.Select(x => x.Split(',')[1].Replace('\"', ' ').Trim())
```

Comment:
I think we should be defensive an not always assume there is a `1` element. I know that is not new in this PR.